src/fides/api/util/tcf/tc_model.py

```diff
@@ -288,7 +299,7 @@ def _build_vendor_consents_and_legitimate_interests(
 
     for vendor_consent in vendor_consents:
         try:
-            int(vendor_consent.id)
+            vendor_consent_id: int = universal_vendor_id_to_id(vendor_consent.id)
```

Comment:
it's not strictly needed for our release increment, since we're not committing to AC support yet. but we'll need to remove AC vendors from compass if we're not supporting them. what would you think of updating this to check for whether the vendor is GVL, based on the prefix? we don't have to make this super robust/clean, but even a rudimentary check would mean that we can feel confident that an AC vendor that sneaks into a fides instance wouldn't make the TC string building here invalid -- to not have to worry about this in the interim until we have proper AC support.

does that make sense? am i understanding things correctly? let me know what you think.

Reply:
thank you, I missed that nuance here in the requirements.

Do you think we should fail right now if there's an "ac." prefixed vendor at the moment or just skip it @adamsachs

Reply:
I'm going to assume an "ac."-prefixed vendor should just be skipped here -

Reply:
@allisonking @adamsachs included a test where there's a system with an AC id but I am excluding it from showing up in the vendor sections of the TC string.  However, its relevant purpose does show up, which I thought was in line with out we treat regular systems currently.  What do you all think?

See test here https://github.com/ethyca/fides/pull/4256/commits/7b86c153cafb9701aef72642d0589badfa802aa1

Reply:
sorry for delay in response - i think what you landed on is good! basically it means we _could_ support an AC vendor if they happen to have a privacy declaration, for whatever reason. that paves the way nicely to the "real" support, which will allow it to show up even without a declaration (the other ticket, not being done here).

@allisonking does this work for you on the FE? i think you already have some AC logic you're building in, so just wanna make sure this won't break anything...

Reply:
> @allisonking @adamsachs included a test where there's a system with an AC id but I am excluding it from showing up in the vendor sections of the TC string. However, its relevant purpose does show up, which I thought was in line with out we treat regular systems currently. What do you all think?

yes, I agree that makes sense! I'm not doing a whole lot specific to AC right now other than generating the string from IDs. any purposes that show up will be treated the same way we treat purposes for GVL/systems 👍

Reply:
wonderful thank you both!